overlord/snapshotstate/snapshotstate.go

```diff
@@ -36,8 +38,15 @@ var (
 	snapstateAll                     = snapstate.All
 	snapstateCheckChangeConflictMany = snapstate.CheckChangeConflictMany
 	backendIter                      = backend.Iter
+
+	// Default expiration time for automatic snapshots, if not set by the user
+	defaultAutomaticSnapshotExpiration = time.Hour * 24 * 31
 )
 
+type snapshotState struct {
+	Expiry time.Time `json:"expiry"`
```

Comment:
Is this the instant when na given snapshot expires? If so perhaps (forgive the bike shed) it should be simply called `ExpiresOn`?

Reply:
Yes, it's the time snapshot expires. Nothing wrong with the name you suggested, but since it has already been ack'ed by Michael and Samuele I'd rather keep it as is. If there is a desire to change it I can also do it in one of the followups (easy to change before the feature is actually used).

Reply:
no very strong opinion on this,  but "expiry-time"  would be more consistent with other similar fields we have